### PR TITLE
Untrusted uploads form

### PR DESCRIPTION
**Pull request by pgrace-google**
i apparently cannot figure out how to take over a PR so making a new one
addressed comments left on https://github.com/google/clusterfuzz/pull/3902 and updated wording, fixed styling/made minor changes

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
OK I think this can proceed.

File: src/appengine/private/components/upload-testcase/upload-form-simplified.html [hunk 1]
```diff
@@ -146,11 +146,11 @@
         </template>
       </div>
       <div class="warning">
-        <b>WARNING</b>: ClusterFuzz doesn't support untrusted workloads.<br>
-        Only upload testcases that you trust enough to run on your own
-        machine. Note that the following jobs do use the Chrome sandbox
-        but this is not a guarantee of protection:<br>
-        [[formatSandboxedJobs(fieldValues.sandboxedJobs)]]
+	      <span><strong>WARNING</strong>: ClusterFuzz only partially supports untrusted workloads.<br></span>
+        ClusterFuzz supports untrusted workloads when the platform is Linux <br>
```
Comment (jonathanmetzman) [inline src/appengine/private/components/upload-testcase/upload-form-simplified.html:150]: nit: can we get rid of the extra space before <br>

File: src/appengine/private/components/upload-testcase/upload-form-simplified.html [hunk 2]
```diff
@@ -146,11 +146,11 @@
...
+        If your workload does not use Linux as the platform, Clusterfuzz will treat it as a trusted workload.
+        Only upload the testcase to ClusterFuzz if you trust it enough to run on your own machine.<br>
```
Comment (jonathanmetzman) [inline src/appengine/private/components/upload-testcase/upload-form-simplified.html:152]: Maybe it needs to be clearer that the second sentence is talking about the same testcase (trusted ones).

**Review by jonathanmetzman — COMMENTED** [review]
lgtm with nits

File: src/appengine/private/components/upload-testcase/upload-form-simplified.html (hunk 1 above)
Comment (pgrace-google) [reply src/appengine/private/components/upload-testcase/upload-form-simplified.html:150]: yeah idk why that happens bc now it looks weird on my editor - but fixed!!

File: src/appengine/private/components/upload-testcase/upload-form-simplified.html (hunk 2 above)
Comment (pgrace-google) [reply src/appengine/private/components/upload-testcase/upload-form-simplified.html]: Changed the wording! lmk if you think that sounds better!

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**